slack-api-client/src/main/java/com/slack/api/audit/impl/AsyncRateLimitExecutor.java

```diff
@@ -80,9 +80,9 @@ public <T extends AuditApiResponse> CompletableFuture<T> execute(
         }, executorService);
     }
 
-    private void initCurrentQueueSizeStatsIfAbsent(String teamId, String methodNameWithSuffix) {
+    private void syncCurrentQueueSizeStats(String teamId, String methodNameWithSuffix) {
```

Comment:
Since this method is a private one, we can safely rename it to represent more accurate behavior.